### PR TITLE
EE-2211 EGAcriptor

### PR DESCRIPTION
**Pull request by afoix**
* Set up CICD in Github actions to make the JAR for the release
* Update README to say to get the JAR from the release
* Add Apache 2.0 license

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,12 @@ E.g. After encryption of **source_file.bam** these files will be generated **sou
 
 # How to run
 Make sure Java 1.8 or above version is installed on your system. type **java -version** and press enter to
-check what java version is installed. Download the **ega-cryptor-2.0.0.jar** and navigate to folder in which jar has been downloaded.
+check what java version is installed. 
+
+- ##### Downloading ega-cryptor
+  Download the **ega-cryptor-<version>.jar** from the [Releases](https://github.com/EbiEga/ega-cryptor/releases) section.
```
Comment (malloryfreeberg) [inline README.md:12]:
```suggestion
  Download the latest **ega-cryptor-<version>.jar** from the [Releases](https://github.com/EbiEga/ega-cryptor/releases) section. 
```

Comment (malloryfreeberg) [reply README.md:12]: I like the change you made here to make it more clear to users :)

**Review by malloryfreeberg — APPROVED** [review]
I reviewed the README.md and LICENSE.txt and made one small suggestion. Otherwise they LGTM.

I did not review the maven yml or pom.xml files - I'll leave that to @bela-juhasz !

**Review by malloryfreeberg — APPROVED** [review]
LGTM!

**Review by bela-juhasz — APPROVED** [review]

**Review by MauricioMoldes — APPROVED** [review]

**Pull request merged**